scripts/api-generator.coffee

```diff
@@ -76,6 +76,23 @@ swagger =
           type: 'string'
           description: 'Error Code'
           example: 'UnauthorizedRequest'
+    ResponseWithToken:
+      type: 'object'
+      properties:
+        ok:
+          type: 'boolean'
+          description: 'If the request processed by endpoint'
+          example: true
+        error:
+          type: 'Error'
+          description: 'Error description'
+          example:
+            message: 'Something went wrong'
+            name: 'SomethingWentWrong'
+        data:
+          type: 'object'
+          description: 'Result of the operation'
+          example: { token: 'some-uuid-string' }
```

Comment:
we don't need to hard code a response if it's only used by one method, instead you can define your example response in place like https://github.com/koding/koding/blob/master/workers/social/lib/social/models/computeproviders/stacktemplate.coffee#L325-L351